_layouts/default.html

```diff
@@ -16,8 +16,10 @@
 
   <title>{{ site.title }}</title>
 
-  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Open+Sans:400,700">
-  <link rel="stylesheet" href="/lib/semantic/css/semantic.min.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:400,700"
```

Comment:
I think we should NOT use SRI for Google Fonts. Apparently CSS they serve is intentionally unversioned and it changes depending on which browser requests it.
See https://github.com/google/fonts/issues/473